tests/goth_tests/test_run_blender.py

```diff
@@ -77,9 +77,9 @@ async def test_run_blender(
             await all_sent.wait_for_result(timeout=120)
             logger.info("All tasks sent")
 
-            await all_computed.wait_for_result(timeout=30)
+            await all_computed.wait_for_result(timeout=120)
```

Comment:
I'm not sure this change is okay, the time between the moment the last task is sent and the moment the last task is computed should not be that long.

Reply:
I know, but it seems that sometimes it takes more than 30s to render the image and send it back. I had to run the test again multiple times without this change.